runtime/compiler/optimizer/IdiomRecognition.cpp

```diff
@@ -6688,10 +6688,33 @@ TR_CISCTransformer::analyzeBoolTable(TR_BitVector **bv, TR::TreeTop **retSameExi
                      takenBV &= tmpBV;
                      ntakenBV -= tmpBV;
                      break;
+
                   default:
-                     TR_ASSERT(false, "not implemented yet");
+                     {
                      // not implemented yet
+                     const char *opName =
+                        TR::ILOpCode((TR::ILOpCodes)n->getOpcode()).getName();
```

Comment:
A very minor comment, but is there any reason why this couldn't be written this way?

```
n->getIlOpCode().getName();
```

Reply:
I ended up not using this because I factored the logic into a function and I wanted to pass the numeric opcode